src/features/lightspeed/lightspeedUser.ts

```diff
@@ -246,7 +246,7 @@ export class LightspeedUser {
       session = await vscode.authentication.getSession(
         useProviderType,
         this.getScopesForAuthProviderType(useProviderType),
-        { createIfNone },
+        { forceNewSession: true },
```

Comment:
I'm not sure it's so simple.

The wizard sends events to and from the view, from and to the controller. If you have multiple tabs open I suspect they'll all receive events unless you begin to filter by/include something like `wizardId`.. I could be wrong.

Reply:
It's a draft/hack, I was trying to understand a problem @anshulbehl is facing.